tests/core/padding-utils/test_padding.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+
+from evm.utils.padding import (
+    pad_left,
+    pad_right,
+    pad32,
+    pad32r
+)
+
+test_31_length_byte_string = b"\x00" * 31
+test_32_length_byte_string = b"\x00" * 32
```

Comment:
These variable names are not great as they don't really make it clear they're storing the padding byte repeated N times. Also, it feels a bit weird to have variables just for two seemingly arbitrary lengths: 31 and 32. Maybe instead you could define a padding_byte variable with the _b"\x00"_ value and then in tests use that with the length multiplier you want